client.go

```diff
@@ -1531,6 +1531,101 @@ func (cl *Client) AddTorrentInfoHash(infoHash metainfo.Hash) (t *Torrent, new bo
 	return
 }
 
+// Save torrent to state file, metadata + pices states.
+func (cl *Client) SaveTorrent(t *Torrent) []byte {
+	t.cl.mu.Lock()
+	defer t.cl.mu.Unlock()
+
+	return nil
```

Comment:
I can't accept an incomplete method for now. There is already Torrent.MetaInfo, and the piece state is dependent on the storage backend, and should be relevant there. File/mmap backends for example do require some state to be added, I've indicated my preferred solution for this here https://github.com/anacrolix/torrent/issues/50#issuecomment-167460940.